packages/ember-glimmer/lib/template.js

```diff
@@ -33,15 +28,22 @@ class Wrapper {
   }
 }
 
+let templateId = 0;
+
 export default function template(json) {
-  let id = templateId++;
+  let id = ++templateId;
 
   let Factory = class extends Wrapper {
     constructor(options) {
-      super(options, id);
-      this.spec = JSON.parse(json);
+      let { env } = options;
+      super(id, env, JSON.parse(json));
+    }
+
+    static create(options) {
+      return new this(options);
```

Comment:
👌 I'm pretty sure this detail is internal to ember-glimmer, so we could refactor it, but we can come back to it once this is fixed